<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSL Creator - AC9HP</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>112</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>QRZ User Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtUserName"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>QRZ Password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtPassword">
       <property name="inputMask">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::PasswordEchoOnEdit</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnLookup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Logon</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtReply">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnQuit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>470</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="txtSearch">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCallsign">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Callsign to lookup:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSession">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>90</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SessionID</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImage">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>471</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>20</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>180</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ADI Log file</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Image file name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnImageBrowse">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnProcess">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>470</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#195fc8;&quot;&gt;Enter your address info and location info&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#195fc8;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>250</width>
      <height>116</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLineEdit" name="txtLine1"/>
    <widget class="QLineEdit" name="txtLine2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtLine3"/>
    <widget class="QLineEdit" name="txtLine4"/>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Info</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtYourCallSign">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Your Call Sign</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtYourCallSignSize">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>50</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Size</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>270</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Location</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtYourCallSignLoc">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>50</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnProcess_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>470</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Test 1</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>50</width>
      <height>116</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLineEdit" name="txtLine1_size"/>
    <widget class="QLineEdit" name="txtLine2_size">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtLine3_size"/>
    <widget class="QLineEdit" name="txtLine4_size"/>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>340</y>
      <width>50</width>
      <height>116</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLineEdit" name="txtLine1_loc"/>
    <widget class="QLineEdit" name="txtLine2_loc">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtLine3_loc"/>
    <widget class="QLineEdit" name="txtLine4_loc"/>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>430</y>
      <width>451</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>910</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQRZ_Lookup_Application">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuQRZ_Lookup_Application"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnQuit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>handleQuit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtSearch</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>handleSearch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLookup</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>handleClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>handleLoad()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>830</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>handleBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnImageBrowse</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>handleImageBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnProcess</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>handleProcess()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>handleSaveDefaults()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>524</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>603</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnProcess_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>handleTestImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>signal1()</signal>
  <slot>handleQuit()</slot>
  <slot>handleClick()</slot>
  <slot>handleSearch()</slot>
  <slot>handleLoad()</slot>
  <slot>handleBrowse()</slot>
  <slot>handleImageBrowse()</slot>
  <slot>handleProcess()</slot>
  <slot>handleSaveDefaults()</slot>
  <slot>handleTestImage()</slot>
 </slots>
</ui>
